api/src/routes/public/status.ts

```diff
@@ -15,25 +14,7 @@ export const statusRoute: FastifyPluginCallbackTypebox = (
   done
 ) => {
   fastify.get('/status/ping', async (req, _reply) => {
-    const url = req.url || 'URL not found';
-    const reqId = req.id || 'REQ_ID not found';
-    const headers = isEmpty(req.headers) ? 'HEADERS not found' : req.headers;
-    const ip =
-      req.headers['x-forwarded-for'] ||
-      req.headers['x-real-ip'] ||
-      req.ip ||
```

Comment:
I do want to debug some of these information from time to time. So a child logger (which adds the request specific annotations automatically).